EntityResolution/code/README.md

```diff
@@ -0,0 +1,130 @@
+
+### **Automata Generator Scripts**
```

Comment:
Can you fix the headers here so that they're the right depth? Maybe put some sort of # Support Code header?